samples/perf-virtuallist-native-with-different-item-size/src/App.js

```diff
@@ -0,0 +1,48 @@
+import Dropdown from '@enact/agate/Dropdown';
+import ThemeDecorator from '@enact/agate/ThemeDecorator';
+import {useCallback, useState} from 'react';
+
+import HorizontalDifferentWidthItemList from './views/HorizontalDifferentWidthItemList';
+import VerticalDifferentHeightItemList from './views/VerticalDifferentHeightItemList';
+import VerticalExpandableDifferentHeightItemList from './views/VerticalExpandableDifferentHeightItemList';
+
+const views = [
+	HorizontalDifferentWidthItemList,
+	VerticalDifferentHeightItemList,
+	VerticalExpandableDifferentHeightItemList
+];
+
+const viewNames = [
+	'HorizontalDifferentWidthItemList',
+	'VerticalDifferentHeightItemList',
+	'VerticalExpandableDifferentHeightItemList'
+];
+
+const defaultViewIndex = 0;
+
+const VirtualListSample = (props) => {
+	const [index, setIndex] = useState(defaultViewIndex);
+
+	const onSelect = useCallback(({selected}) => {
+		setIndex(selected);
+	}, []);
+
+	const View = views[index];
+
+	return (
+		<div {...props}>
+			<Dropdown
```

Comment:
this dropdown doesn't get the default value on application load. can you check please?
![image](https://user-images.githubusercontent.com/63335068/190400282-9be790bc-e170-41df-87f2-1cc2fc95ff72.png)